src/App.tsx

```diff
@@ -1,18 +1,34 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
```

Comment:
It is recommended to keep the original mapping format for items, which is 'Item ${n}' instead of just '${n}'.